src/Sarif.FunctionalTests/ConverterTestData/FxCop/FxCopExampleReport2.xml.sarif

```diff
@@ -55,14 +55,14 @@
           }
         },
         {
+          "analysisTarget": {
```

Comment:
As above: `location.physicalLocation`, not `result.analysisTarget`.

Reply:
Do this here and throughout the FxCop test files. This is related to the code change we just discussed on the phone.